### PR TITLE
Fixed Regression where ETags which had quotes surrounding the value, …

### LINKED ISSUE #22877: [BUG] Etag Quotes missing v12.9.0+ UploadAsync
**Describe the bug**
When I upgrade from v12.8.4 to 12.9.0 or 12.9.1 the etag quotes are gone.  I am using blobClient.UploadAsync. I checked the version of the request and it is set to Version 2020-08-04.

**Expected behavior**
Quotes to be included on etag  using blobContentInfo.Value.ETag.ToString()

**Actual behavior (include Exception or Stack Trace)**
Etag returns without quotes.

**To Reproduce**
Steps to reproduce the behavior (include a code snippet, screenshot, or any additional information that might help us reproduce the issue)

1. change azure.storage.blob version to 12.9.1. Upload document "await blobClient.UploadAsync(dataStream)".
2. check etag value  "string eTag = blobContentInfo.Value.ETag.ToString()"

**Environment:**
 - Name and version of the Library package used: [e.g. Azure.Storage.Blobs 12.9.1] 
 - Hosting platform or OS and .NET runtime version (`dotnet --info` output for .NET Core projects): [.NET 5.0 Console Application
 - IDE and version : [e.g. Visual Studio 16.10.2]

### PR DESCRIPTION
**Pull request by amnguye**
…were removed

Resolves https://github.com/Azure/azure-sdk-for-net/issues/22877

### CODE REVIEW

File: sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <DefineConstants>DataLakeSDK;$(DefineConstants)</DefineConstants>
     <PackageTags>Microsoft Azure Storage Files;Microsoft;Azure;File;Files;Data Lake;Storage;StorageScalable;$(PackageCommonTags)</PackageTags>
     <Description>
-      This client library enables working with the Microsoft Azure Storage File service for storing binary and text data.
+      This client library enables working with the Microsoft Azure Storage Data Lake File service for storing binary and text data.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj:13]: I searched DataLake through nuget on VS and we weren't the first ones to show up cause we didn't have Data Lake in the description. Just added this so the search algorithm hopefully picks us up first

File: sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md [hunk 2]
```diff
@@ -33,7 +33,7 @@
 ## 12.7.0-beta.4 (2021-05-12)
 - Added support for service version 2020-08-04.
 - Added support for Share and Share Snapshot Lease.
-- DataLakeLeaseClient now remembers the Lease ID after a lease change.
+- ShareLeaseClient now remembers the Lease ID after a lease change.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md:36]: Let me know if I can't update an older change log thing, I was just looking for something and I couldn't find it cause of this typo.

File: sdk/storage/Azure.Storage.Files.Shares/tests/FileClientTests.cs [hunk 3]
```diff
@@ -3161,6 +3193,24 @@ public async Task ChangeLeaseAsync_Error()
                 e => Assert.AreEqual("ResourceNotFound", e.ErrorCode));
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = ShareClientOptions.ServiceVersion.V2020_02_10)]
+        public async Task RenewLeaseAsync_Error()
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.Shares/tests/FileClientTests.cs:3198]: I didn't see a test for this exception, so I added it here before I forget to it.
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Files.Shares/src/ShareLeaseClient.cs#L1027
Let me know if there's already an existing test. I know we probably would have to remove this test if they eventually add in renew for Share Files.

File: sdk/storage/Azure.Storage.Blobs/CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,7 @@
...
 ## 12.10.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed bug / regression where the ETag values that had quotation marks surrounding it were being removed in versions 12.9.0, 12.9.1, 12.10.0-beta.1 and 12.10.0-beta.2.
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/CHANGELOG.md:4]: How about "...were being removed starting in version 12.9.0"

File: sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,7 @@
...
 ## 12.8.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed bug / regression where the ETag values that had quotation marks surrounding it were being removed in versions 12.7.0, 12.8.0-beta.1 and 12.8.0-beta.2.
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md:4]: Same

File: sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,7 @@
...
 ## 12.8.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed bug / regression where the ETag values that had quotation marks surrounding it were being removed in versions 12.7.0, 12.8.0-beta.1 and 12.8.0-beta.2.
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md:4]: Same

**Review by seanmcc-msft — APPROVED** [review]
Looks good.  Are we sure we addressed this for all applicable APIs?

**Comment by seanmcc-msft** [comment]
Looks like tests aren't passing.

**Review by kasobol-msft — APPROVED** [review]

**Pull request merged**